diff --git a/external/loki-mq/CMakeLists.txt b/external/loki-mq/CMakeLists.txt
index 68926f8..21d5544 100644
--- a/external/loki-mq/CMakeLists.txt
+++ b/external/loki-mq/CMakeLists.txt
@@ -76,7 +76,7 @@ target_link_libraries(oxenmq PRIVATE Threads::Threads)
 set(oxenmq_build_static_libzmq OFF)
 if(TARGET libzmq)
     target_link_libraries(oxenmq PUBLIC libzmq)
-elseif(BUILD_SHARED_LIBS)
+else()
     include(FindPkgConfig)
     pkg_check_modules(libzmq libzmq>=4.3 IMPORTED_TARGET)
 
@@ -95,8 +95,6 @@ elseif(BUILD_SHARED_LIBS)
     else()
         set(oxenmq_build_static_libzmq ON)
     endif()
-else()
-    set(oxenmq_build_static_libzmq ON)
 endif()
 
 if(oxenmq_build_static_libzmq)
